### PR TITLE
Add Personalization to welcome message

### PR DESCRIPTION
**Pull request by DaveAnswer**
Add Personalization to welcome message , if Skills Personalization is enabled form the permission tab.

If not applicable default value is returned.

### CODE REVIEW

File: lambda/src/personalizationUtil.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+     return `'<alexa:name type="first" personId="${getPersonId(handlerInput)}" />`
+ }
+ 
+ /**
+  * Export the list of needed for clients to use
+  **/
+ module.exports = {
+     getPersonalizedPrompt,
+     getPerson
+ };
```
Comment (Shreyas-vgr) [inline lambda/src/personalizationUtil.js]: nit: Add EOL

File: lambda/src/personalizationUtil.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ /**
+  * Handle fallback logic incase person not found.
+  */
+ const handleFallback = () => { return EMPTY }
+ 
+ /**
+  * Get personalized greeting from person Id.
+  * Refer: https://developer.amazon.com/en-US/docs/alexa/custom-skills/add-personalized-greetings-or-prompts.html
+  */
+ const getPersonalizedPromptFromId = handlerInput => {
```
Comment (Shreyas-vgr) [inline lambda/src/personalizationUtil.js]: should this be a function too 
```js
const getPersonalizedPromptFromId = (handlerInput) => { ...}

```

Comment (DaveAnswer) [reply lambda/src/personalizationUtil.js]: Not necessarily , as this method is not exposed outside the Util and does not call any async API.

File: lambda/src/personalizationUtil.js (hunk 1 above)
Comment (DaveAnswer) [reply lambda/src/personalizationUtil.js]: done

**Review by narayanahari — APPROVED** [review]
Change looks good to me.

**Review by Shreyas-vgr — APPROVED** [review]

**Pull request merged**